src/Widget/ItemList.php

```diff
@@ -31,7 +33,7 @@ class ItemList extends BaseHtmlElement
     /** @var string */
     private $itemLayoutClass = ItemLayout::class;
 
-    /** @var ?string Message to show if the list is empty */
+    /** @var ?ValidHtml Message to show if the list is empty */
     protected $emptyStateMessage;
```

Comment:
I'm thinking about renaming it to `$emptyStateContent` including the getters/setters.  What is your opinion @nilmerg ?

Reply:
I'd rather not change the method name. Thus neither the property name.